pkgs/top-level/python-aliases.nix

```diff
@@ -0,0 +1,37 @@
+lib: pythonPackages:
```

Comment:
You should probably use a `lib: self: super:` type of setup as well like `top-level/aliases.nix` or we'll run in trouble down the line if we want to do more complicated stuff in this file to ensure backwards compatibility like overriding.